.circleci/config.yml

```diff
@@ -2413,11 +2413,11 @@ workflows:
          requires:
            - package-test-deb
          <<: *master_and_release_only
-     - ami-tests-development:
-          context: scalyr-agent
-          requires:
-            - build-windows-package
-            - build-linux-packages-and-installer-script
+#     - ami-tests-development:
```

Comment:
Why did you comment these out?  Add a TODO to explain that when when we could add the tests back in.

Reply:
Sorry, those lines are become commented every time after I merge master into the branch and I miss that constantly..